misk-feature/src/main/kotlin/misk/feature/DynamicFlags.kt

```diff
@@ -0,0 +1,31 @@
+package misk.feature
+
+/**
+ * Interface for evaluating dynamic flags. Dynamic flags are similar to feature flags, but they
+ * don't support different variations for different keys.
+ */
+interface DynamicFlags {
```

Comment:
I don't love the naming. We already have a concept of DynamicFlags. Could we not introduce a different name for this to avoid confusion?

Reply:
This is meant to be a convenience class to make feature flags compatible with dynamic flags. What do you suggest?

Reply:
`DynamicConfig`?

Reply:
DynamicConfig it is then